### PR TITLE
Master PG12 TEST

### PR DESCRIPTION
**Pull request by jrgemignani**
DO NOT MERGE!!!
DO NOT MERGE!!!

This PR is a test, for bringing the master up to PG12, to allow others to review the work.

### CODE REVIEW

**Comment by jrgemignani** [comment]
If you can review this when you have time. This way I can correct any oversights or mistakes before going forward.

**Comment by jrgemignani** [comment]
I have updated the PR contents; some files will have changed.

File: src/backend/executor/cypher_create.c
Comment (dehowef) [inline src/backend/executor/cypher_create.c:1]: Line 109 introduced a typo. Aquire should be acquire.

File: src/backend/executor/cypher_merge.c
Comment (dehowef) [inline src/backend/executor/cypher_merge.c:1]: Line 120, same correction as mentioned above -- aquire-->acquire

File: src/backend/nodes/cypher_readfuncs.c
Comment (dehowef) [inline src/backend/nodes/cypher_readfuncs.c:1]: I think John mentioned this too, but I will add my concern. Should there be a consistency in the READ_FIELD function for oid? Right now there are two changed instances of READ_OID_FIELD --> READ_INT_FIELD and one of READ_OID_FIELD to READ_UINT_FIELD.

File: src/include/parser/cypher_kwlist.h
Comment (dehowef) [inline src/include/parser/cypher_kwlist.h:1]: Lack of EOL in this file. Maybe needs one

**Review by dehowef — COMMENTED** [review]
This is an unfinished review-- The rest of my comments will be posted on #1015.

**Pull request closed without merging**